config/initializers/date_formats.rb

```diff
@@ -1 +1,2 @@
 Date::DATE_FORMATS[:nomis] = '%Y-%m-%dT%H:%M:%S'
+Time::DATE_FORMATS[:nomis] = '%Y-%m-%dT%H:%M:%S'
```

Comment:
it looks a bit confusing having :nomis for date and time

Reply:
I'm not sure of a better name, though? Happy to take suggestions 🙂

Reply:
maybe `nomis_time` 
`nomis_full_time`, `nomis_extended_time`,  ... :)

Reply:
Hi @alexdesi I think its OK as is:
```
Date.today.to_s(:nomis)
Time.zone.to_s(:nomis)
```

Reply:
Going with as is as this is currently a bug in production.

Reply:
ok, np, one point for me was it would be simpler to search in your editor for the time key :nomis would not gonna help you